src/nexmo/__init__.py

```diff
@@ -633,7 +654,7 @@ def _jwt_signed_get(self, request_uri, params=None):
 
         return self.parse(
             self.api_host,
-            self.session.get(uri, params=params or {}, headers=self._headers()),
+            requests.get(uri, params=params or {}, headers=self._headers()),
```

Comment:
why is this changing from self.session to requests?

Reply:
reverted

Reply:
can you check if it really was reverted? I still see `self.session` replaced by `requests`